instrumentation-api/src/test/groovy/io/opentelemetry/instrumentation/api/tracer/BaseTracerTest.groovy

```diff
@@ -16,14 +16,12 @@
 
 package io.opentelemetry.instrumentation.api.tracer
 
-import io.opentelemetry.auto.test.utils.ConfigUtils
 import io.opentelemetry.auto.util.test.AgentSpecification
 import io.opentelemetry.trace.Span
-import io.opentelemetry.trace.attributes.SemanticAttributes
 import spock.lang.Shared
 
+// TODO add tests for BaseTracer
```

Comment:
If there are any tests that came to mind when you wrote this TODO, I think it's better to file an issue with some examples. I have no idea how to resolve this TODO right now :P

Reply:
see #1063